src/classic/clvm_tools/sha256tree.rs

```diff
@@ -1,7 +1,28 @@
+use std::fmt::Display;
+
 use clvm_rs::allocator::{Allocator, NodePtr, SExp};
 
 use crate::classic::clvm::__type_compatibility__::{sha256, Bytes, BytesFromType};
 
+#[derive(Clone, Debug, PartialEq, Eq, Hash)]
+pub struct TreeHash {
+    s: String,
```

Comment:
eventually I would expect this to be a `pub struct TreeHash([u8; 32])`

Reply:
ya makes sense